practice-two/src/components/Toast/Toast.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useRef } from "react";
+import "./Toast.css";
+import PropTypes from "prop-types";
+
+const Toast = ({ message, backgroundColor }) => {
+  const toastContainerRef = useRef(null);
+
+  useEffect(() => {
+    if (toastContainerRef.current) {
+      const container = toastContainerRef.current;
+
+      setTimeout(() => {
+        container.classList.add("toast__close");
```

Comment:
why the first time need to close toast message?

Reply:
I want to show some important notifications first and then close them after a fixed time to attract user's attention. Then I can show other toast.